.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+# Description
+
+<!-- summary of changes (add screenshots/videos to the bottom of the PR if applicable)
+- relevant context from Slack is summarized and included -->
+
+<!-- if this is a bug fix, include a link to the issue (e.g. `closes #123`), and how to reproduce the bug on master
+if this is a refactor/feature, why this change was made -->
+
+# Testing
+
+<!-- how you tested your changes
+
+- if this is a UI change, mention what accessibility testing has been done
+- if applicable, include unit tests for complex functionality / behavior modifications -->
+
+<!-- if applicable, include instructions for how the reviewer can test the changes -->
+
+<!-- in a self-review of the code, point out:
+
+  - explanations for code/design decisions that may confuse reviewers
+  - important/core bits of the PR
+  - rationale for design decisions
+  - items that may need further discussion
+
+Thank you for contributing to PrairieLearn! -->
```

Comment:
I added this to be friendly to our open-source contributors, though we might not want it if it adds noise?